test/compress/issue-1443.js

```diff
@@ -0,0 +1,68 @@
+// these tests assume local variable "undefined" has undefined as value
```

Comment:
Could this be rephrased as:
```
// tests assume that variable `undefined` not redefined and has `void 0` as value
```
Also, please put an empty line after this comment.